### DISCUSSION TITLE
Can we get a tutorial for creating a basic item?

### DISCUSSION THREAD

**Discussion by pixelstuff** [ikas-mc/ContextMenuForWindows11#183] [General]
I installed the app, turned it on, created a new menu item and saved it, but nothing new is showing up in the context menu. Can we get a tutorial on how to get a basic item going? My assumption is that even just the Title and EXE field should be enough to see something, even if it was a non-functional entry. Do we have to reboot the computer, or what else is needed?

**Comment by tboss3000** [comment]
Dam I just bought the app and its so confusing even where to start

**Comment by ikas-mc** [comment]
sorry, but you can try to import examples

**Reply by tocguy** [reply]
Again, howwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww?

**Comment by ikas-mc** [comment]
https://github.com/ikas-mc/ContextMenuForWindows11/tree/main/menuSample

**Reply by tocguy** [reply]
I find no means of loading a sample. Howwwwwwwwwwwwwwwwww?

**Reply by ikas-mc** [reply]
> I find no means of loading a sample. Howwwwwwwwwwwwwwwwww?

@tocguy 

new samples here : https://github.com/ikas-mc/ContextMenuForWindows11/discussions/categories/menus

1. copy json
``` json
{
  "title": "Debug",
  "exe": "customeContextMenuDebug.exe",
  "param": "{path}",
  "icon": "",
  "acceptDirectory": true,
  "acceptDirectoryFlag": 15,
  "acceptFile": false,
  "acceptFileFlag": 4,
  "acceptExts": "",
  "acceptFileRegex": "",
  "acceptMultipleFilesFlag": 2,
  "pathDelimiter": ",",
  "paramForMultipleFiles": "{path}",
  "index": 2,
  "showWindowFlag": 0
}

```
2. open app
3. click + to add a new menu
4. click Paste
5. click Save

**Reply by tocguy** [reply]
TY
